livekit-plugins/livekit-plugins-turn-detector/livekit/plugins/turn_detector/base.py

```diff
@@ -121,6 +121,7 @@ def __init__(
         self,
         model_type: EOUModelType = "en",  # default to smaller, english-only model
         inference_executor: InferenceExecutor | None = None,
+        custom_threshold: float | None = None,
```

Comment:
```suggestion
        unlikely_threshold: float | None = None,
```
also some docs about the use? i.e. it'll default to what's trained